README.md

```diff
@@ -42,3 +42,26 @@ Access Layer5 Cloud at https://meshery.layer5.io
 ## About Layer5
 
 Layer5's cloud native application and infrastructure management software enables organizations to expect more from their infrastructure. We embrace _developer_-defined infrastructure. We empower developers to change how they write applications, support _operators_ in rethinking how they run modern infrastructure, and enable _product owners_ to regain full-control over their product portfolio.
+
+### Directory structure
+1. `action-assets`: 
+      1.   Store the snapshots taken as part of pull requests. Each snapshot have 2 versions and are namespaced based on the year and month they were taken, for eg: snapshot against a particular PR created on 10th July 2024 will be stored at
+      a. Light: `action-assets/2024@7/<design-id>-light.png` 
+      b. Dark version: `action-assets/2024@7/<design-id>-dark.png` 
+ 
+      2. `design-assets`: Stores the snapshots taken for catalog items. Every time a catalog item is updated the corresponding snapshot for that catalog also gets updated. We refer the catalog snaphosts from different locations and hence require a permanenet link.
```

Comment:
@MUzairS15, this is only done for published or for published and public items?

Reply:
catalog items: i.e. only published